package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "rabbit-queue",
-  "version": "5.1.0",
+  "version": "5.2.0",
```

```suggestion
  "version": "5.1.0",
```
We will bump the version before we publish a new version.